src/gluonts/dataset/artificial/__init__.py

```diff
@@ -19,6 +19,7 @@
     RecipeDataset,
     constant_dataset,
     default_synthetic,
+    shuffle_testing_dataset,
```

Comment:
We don't need to have this in gluonts.dataset.artificial, I think

Reply:
just removed this